cypress/integration/individual-pages/activity.spec.ts

```diff
@@ -1,27 +1,287 @@
-import { asAMaritimePleasureBeaconOwner } from "../common/i-can-enter-beacon-information.spec";
+import { PageURLs } from "../../../src/lib/urls";
 import {
-  andIHaveEnteredNoInformation,
-  andIHaveSelected,
+  andIClickContinue,
+  givenIHaveACookieSetAndIVisit,
+  givenIHaveSelected,
   requiredFieldErrorMessage,
   thenIShouldSeeAnErrorMessageThatContains,
   thenIShouldSeeAnErrorSummaryLinkThatContains,
+  thenIShouldSeeFormErrors,
   thenMyFocusMovesTo,
   whenIClickContinue,
   whenIClickOnTheErrorSummaryLinkContaining,
+  whenIType,
 } from "../common/selectors-and-assertions.spec";
 
 describe("As a beacon owner, I want to submit the primary activity for my beacon", () => {
   const otherActivitySelector = "#other-activity";
+  const otherActivityTextRequiredErrorMessage = [
+    "Enter a description",
+    "activity",
+  ];
 
-  it("displays an error if no activity is selected", () => {
-    asAMaritimePleasureBeaconOwner();
-    andIHaveEnteredNoInformation();
-    whenIClickContinue();
-    thenIShouldSeeAnErrorMessageThatContains(requiredFieldErrorMessage);
-    thenIShouldSeeAnErrorSummaryLinkThatContains(
-      "Activity",
-      requiredFieldErrorMessage
-    );
+  beforeEach(() => {
+    givenIHaveACookieSetAndIVisit(PageURLs.environment);
+  });
+
+  describe("As a maritime, pleasure owner", () => {
```

Comment:
Added additional tests around the validation for the different combinations of:

* Maritime pleasure/commercial
* Aviation pleasure/commercial
* Land

Reply:
As discussed, raised this on the backlog to update the directory structure to rename test file names to describe the behaviour rather than the page.

See:  https://github.com/mcagov/beacons-webapp/issues/231